src/types/index.ts

```diff
@@ -131,6 +131,8 @@ type Destination = {
   WorkspaceID: string;
   Transformations: UserTransformationInput[];
   RevisionID?: string;
```

Comment:
Is rudder-server sending these 2 keys to rudder-transformer as of today in production ?

Reply:
yes, confirmed this with @yashasvibajpai , but not being used  in transformer